conftest.py

```diff
@@ -147,6 +147,25 @@ def default_configuration():
         enable_unsafe_features=True,
         use_insecure_key_cache=True,
         insecure_key_cache_location="ConcreteNumpyKeyCache",
+        fhe_simulation=False,  # Simulation compilation is done lazilly on circuit.simulate
```

Comment:
off-topic ; Will an error be raised if both fhe_simulation and fhe_execution flags are set to true?

Reply:
> off-topic ; Will an error be raised if both fhe_simulation and fhe_execution flags are set to true?

No you can set both to true. By default, the simulation one is off but if you trigger circuit.simulate, it will compile for simulation.



> Maybe add a comment, saying that the default config is fhe_execution in this docstring function

Actually default configuration is both of them.

Reply:
> Actually default configuration is both of them.

what do you mean ? in CP (and here) fhe simu is not enabled by default right ?